spec.md

```diff
@@ -80,85 +80,97 @@ A CO SHOULD be equipped to handle both centralized and headless plugins, as well
+|                  |                              |
+|                  |        +------------------+  |
+|                  |        | Group Controller |  |
+|                  +-------->     Plugin       |  |
+|                           +------------------+  |
+|                                                 |
++-------------------------------------------------+
 
 Figure 2: Headless Plugin deployment, only the CO Node hosts run
-Plugins. Separate, split-component Plugins supply the Controller
-Service and the Node Service respectively.
+Plugins. Separate, split-component Plugins supply the Node Service,
+the Controller Service and the Group Controller Service (optional)
+respectively.
 ```
 
 ```
-                            CO "Node" Host(s)
-+-------------------------------------------+
-|                                           |
-|  +------------+           +------------+  |
-|  |     CO     |   gRPC    | Controller |  |
-|  |            +----------->    Node    |  |
-|  +------------+           |   Plugin   |  |
-|                           +------------+  |
-|                                           |
-+-------------------------------------------+
+                                  CO "Node" Host(s)
++-------------------------------------------------+
+|                                                 |
+|                           +------------------+  |
+|  +------------+           |      Node        |  |
+|  |     CO     |   gRPC    |   Controller     |  |
+|  |            +-----------> Group Controller |  |
+|  +------------+           |     Plugin       |  |
+|                           +------------------+  |
+|                                                 |
++-------------------------------------------------+
 
 Figure 3: Headless Plugin deployment, only the CO Node hosts run
-Plugins. A unified Plugin component supplies both the Controller
-Service and Node Service.
+Plugins. A unified Plugin component supplies both the Node Service,
+the Controller Service and the Group Controller Service (optional).
 ```
 
 ```
-                            CO "Node" Host(s)
-+-------------------------------------------+
-|                                           |
-|  +------------+           +------------+  |
-|  |     CO     |   gRPC    |    Node    |  |
-|  |            +----------->   Plugin   |  |
-|  +------------+           +------------+  |
-|                                           |
-+-------------------------------------------+
+                                  CO "Node" Host(s)
++-------------------------------------------------+
+|                                                 |
+|  +------------+           +------------------+  |
+|  |     CO     |   gRPC    |       Node       |  |
+|  |            +----------->      Plugin      |  |
+|  +------------+           +------------------+  |
+|                                                 |
++-------------------------------------------------+
```

Comment:
Is it necessary to update this drawing?

Reply:
No, this change is only for the unified width as same as the other in this chapter.